### ISSUE TITLE
Filter process table

### ISSUE THREAD

**Issue by cc-a** [ImperialCollegeLondon/dune_processes#94]
See #85 for reference.

This issues covers only a basic version of this functionality where the user enters data into the filter form and submits it (via dedicated button or hitting return) and the whole page reloads with the filtered data. Auto-updating the table as the user types will be considered in another issue.

Implementation would ideally use [https://django-filter.readthedocs.io/en/stable/index.html](https://django-filter.readthedocs.io/en/stable/index.html) however this seems to only work with data stored in models. We should get a view on #93 first as that may require a move to storing process data in models.

**Comment by dalonsoa** [comment]
To do after #93

**Comment by dalonsoa** [comment]
Let's move on without models.

**Comment by dalonsoa** [comment]
This is what I'll plan do use: https://taskbadger.net/blog/tables.html#bonus-2-filtering

**Comment by cc-a** [comment]
Closed by #153
